src/OrchardCore.Modules/OrchardCore.Taxonomies/Views/TermItem.cshtml

```diff
@@ -0,0 +1,23 @@
+@model dynamic
+@{
+    TagBuilder tag = Tag(Model, "li");
+
+    // Morphing the shape to keep Model untouched
+    Model.Metadata.Alternates.Clear();
+    Model.Metadata.Type = "TermContentItem";
+
+    tag.InnerHtml.AppendHtml(await DisplayAsync(Model));
```

Comment:
Not obvious that using InnerHtml is necessary. Looks like it could just work with some direct calls to `@await displayAsync`. Unless you found a reason due to ordering that was not possible.

Reply:
No reason. I just hadn't done it that way. Have removed the tagbuilder and done directly with @await DisplayAsync()